src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_ActiveFareProductsTabScreen.tsx

```diff
@@ -92,18 +141,20 @@ export const TicketTabNav_ActiveFareProductsTabScreen = ({
           )}
 
           {hasSentFareContracts && (
-            <LinkSectionItem
-              text={t(TicketHistoryModeTexts.sent.title)}
-              accessibility={{
-                accessibilityHint: t(TicketHistoryModeTexts.sent.titleA11y),
-              }}
-              testID="sentToOthersButton"
-              onPress={() =>
-                navigation.navigate('Ticketing_TicketHistoryScreen', {
-                  mode: 'sent',
-                })
-              }
-            />
+            <View ref={sentFareContractRef} collapsable={false}>
```

Comment:
Adding this `View` breaks the section, and won't give bottom rounding. The `Section` should only contain `SectionItem`s as children.